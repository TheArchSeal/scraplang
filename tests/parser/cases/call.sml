x[0];
x();
x(0, 0, 0);
x {};
x { 0, 0, 0 };
x.y;
x.y.z.w;
