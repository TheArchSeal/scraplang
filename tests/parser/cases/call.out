stmt (3):1:1 ;
    expr (9):1:1 []
        expr (3):1:1 x
        expr (3):1:3 0
stmt (3):2:1 ;
    expr (10):2:1 ()
        expr (3):2:1 x
stmt (3):3:1 ;
    expr (10):3:1 ()
        expr (3):3:1 x
        expr (3):3:3 0
        expr (3):3:6 0
        expr (3):3:9 0
stmt (3):4:1 ;
    expr (11):4:1 {}
        expr (3):4:1 x
stmt (3):5:1 ;
    expr (11):5:1 {}
        expr (3):5:1 x
        expr (3):5:5 0
        expr (3):5:8 0
        expr (3):5:11 0
stmt (3):6:1 ;
    expr (12):6:1 .y
        expr (3):6:1 x
stmt (3):7:1 ;
    expr (12):7:1 .w
        expr (12):7:1 .z
            expr (12):7:1 .y
                expr (3):7:1 x
